ksql-functional-tests/src/test/resources/expected_topology/0_6_0-pre/arraycontains_-_filter_rows_where_the_ARRAY_column_contains_a_specified_STRING

```diff
@@ -51,14 +51,14 @@ Topologies:
       --> KSTREAM-FILTER-0000000002
       <-- KSTREAM-SOURCE-0000000000
     Processor: KSTREAM-FILTER-0000000002 (stores: [])
-      --> KSTREAM-MAPVALUES-0000000003
+      --> SELECT-0
```

Comment:
it's kinda weird to have just some of the mapValues have custom names... I suppose we can handle the rest in follow-up PRs

Reply:
I've more PRs coming in a similar vain. Any left unchanged will be picked up in #3656.